.gitignore

```diff
@@ -67,3 +67,5 @@ dump.rdb
 *.swn
 *.swm
 .vimrc
+
+client/config.js
```

Comment:
and how the environment keys are set in this process?
I want to allow travis to work on forks but for that I still need the `env` variables.